openfisca_france/parameters/marche_travail/salaire_minimum/smic/smic_b_mensuel.yaml

```
@@ -236,9 +236,11 @@ values:
     value: 1645.58
   2022-08-01:
     value: 1678.95
+  2023-01-01:
+    value: 1709.28
 metadata:
   ux_name: SMIC brut mensuel
-  description_en: Minimum wage - SMIC (1970-2017)
+  description_en: Minimum wage - SMIC (1970+)
```

```suggestion
  description_en: Minimum wage - SMIC (1970 -)
```
Pour être omogène avec les autres entrées même si je reconnais que ce n'est pas joli.
Mais on va essayer de retirer ces intervalles à terme pour les recalculer.